cmake/FlowCommands.cmake

```diff
@@ -27,7 +27,7 @@ macro(actor_compile target srcs)
       string(REPLACE ".actor.cpp" ".actor.g.cpp" tmp ${src})
     endif()
     set(actor_compiler_flags "")
-    if(ACTOR_COMPILE_DISABLE_ACTOR_WITHOUT_WAIT)
+    if(ACTOR_COMPILE_DISABLE_ACTOR_WITHOUT_WAIT_WARNING)
```

Comment:
The line just below this one has the old flag name

Reply:
Oops. Fixed.